src/main/java/bio/terra/cloudfiletodatastore/FileUploadedMessage.java

```diff
@@ -0,0 +1,72 @@
+package bio.terra.cloudfiletodatastore;
```

Comment:
I'm sure we'll need to add more information to this class especially if it's shared between TestRunner and DL